src/charm_state.py

```diff
@@ -167,7 +167,7 @@ class SynapseConfig(BaseModel):  # pylint: disable=too-few-public-methods
 
     allow_public_rooms_over_federation: bool = False
     enable_irc_bridge: bool = False
-    irc_bridge_admins: str | None = Field(None, regex=r"(@\w+:\w+\.\w+,?)+")
+    irc_bridge_admins: str | None = Field(None, regex=r"(\w+:\w+\.\w+,?)+")
```

Comment:
Why the "@" was removed?
While adding the publish_room_allowlist, I submitted a refactor that affects the field too since both are user id list.
https://github.com/canonical/synapse-operator/pull/315/files#diff-903f257e0f0168353a1da3fc6c6df85e19955467e7c78f584d00a481b7af57a8R230

Reply:
The field was removed because juju interprets a leading `@` sign as pointing to a file location (and the action fails) - now you pass in the admins without a leading `@` and that is prepended when constructing the list of admins.

Reply:
Wow :( Now I remember that you mentioned something about it ...
I'll change that in my PR too then. Thanks.